notebooks/220-cross-lingual-books-alignment/220-cross-lingual-books-alignment.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<p>&gt;&gt;The notebook also shows how to accelerate the process of getting vectors from sentences using the OpenVINO™ framework. </p><p> </p><p>sound like some secondary and unimportant  or optional, however it is the main goal of the notebook. Could you please rephrase</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Agree. Rewrite to
> The notebook shows how to accelerate the most computationally expensive part of the pipeline - getting vectors from sentences - using the OpenVINO™ framework.